pkgs/top-level/all-packages.nix

```diff
@@ -2165,8 +2165,8 @@ with pkgs;
 
   ares = callPackage ../applications/emulators/bsnes/ares { };
 
-  bsnes-hd = callPackage ../applications/emulators/bsnes/bsnes-hd {
-    inherit (darwin.apple_sdk.frameworks) Cocoa OpenAL;
+  bsnes-hd = darwin.apple_sdk_11_0.callPackage ../applications/emulators/bsnes/bsnes-hd {
+    inherit (darwin.apple_sdk_11_0.frameworks) Cocoa OpenAL;
```

Comment:
Do that same "internalization" as in ares code.

Reply:
Done.